### DISCUSSION TITLE
Why is the `bgLayoutImgList` prop nowhere in the documentation?

### DISCUSSION THREAD

**Discussion by cliancarlos** [ant-design/pro-components#6837] [Q&A]
Hello everyone, I'm new to Ant Design and I'm studying hard to be able to work with it on a daily basis, and it's been great.
But I was trying to make[ **this design**](https://prnt.sc/ic5cV1E12-K8), I managed it easily using the **Layout/>** component with CSS, but I wanted to do it in **ProLayout/>** which with CSS was being a little more work, I looked at all the api/tokens options I didn't find anything that solved the problem, but after investigating each of the demos, luckily I found this prop `bgLayoutImgList`, tested it, and it did what I wanted brilliantly.

But the question that remains for me is, why is this prop not in the documentation? or maybe it's somewhere but I didn't find it?

Thanks.

**Comment by arvinxx** [comment]
maybe you should ask the question in pro-components repo.

**Reply by cliancarlos** [reply]
Really, sorry, I just didn't notice. Thank you for transferring.
